src/components/transactions/[txid]/TransactionSummaryTable.tsx

```diff
@@ -0,0 +1,66 @@
+import { Transaction, TransactionVin, TransactionVout } from '@defichain/whale-api-client/dist/api/transactions'
+import { useSelector } from 'react-redux'
+import { RootState } from '@store/index'
+import { AdaptiveList } from '@components/commons/AdaptiveList'
+import { Link } from '@components/commons/Link'
+import { format, fromUnixTime } from 'date-fns'
+import BigNumber from 'bignumber.js'
+
+export function TransactionSummaryTable (props: { transaction: Transaction, vins: TransactionVin[], vouts: TransactionVout[], fee: BigNumber }): JSX.Element {
+  const transaction = props.transaction
+  const vins = props.vins
+  const vouts = props.vouts
+  const fee = props.fee
+
+  return (
+    <div className='mt-5 flex flex-col space-y-6 items-start lg:flex-row lg:space-x-8 lg:space-y-0'>
+      <SummaryTableListLeft transaction={transaction} vins={vins} vouts={vouts} fee={fee} />
+      <SummaryTableListRight transaction={transaction} vins={vins} vouts={vouts} fee={fee} />
+    </div>
+  )
+}
+
+function SummaryTableListLeft (props: { transaction: Transaction, vins: TransactionVin[], vouts: TransactionVout[], fee: BigNumber }): JSX.Element {
+  const { count: { blocks } } = useSelector((state: RootState) => state.stats)
```

Comment:
In the future, I think we can move this API so that it's server-side render and always available. Hydrate into store after hitting SPA.